plugins/doc_fragments/acme.py

```diff
@@ -33,7 +33,8 @@ class ModuleDocFragment(object):
          key."
       - "RSA keys can be created with C(openssl genrsa ...). Elliptic curve keys can
          be created with C(openssl ecparam -genkey ...). Any other tool creating
-         private keys in PEM format can be used as well."
+         private keys in PEM format can be used as well - e.g. L(openssl_privatekey
+         module,https://docs.ansible.com/ansible/latest/modules/openssl_privatekey_module.html)."
```

Comment:
I would change the text so that openssl_privatekey comes first, and using the `openssl` binary is only mentioned as a fallback:
```
      - "Private keys can be created with the M(community.crypto.openssl_privatekey) module.
        If the requisites (pyOpenSSL or cryptography) are not available, keys can also be created
        directly with the C(openssl) command line tool: RSA keys can be created with
        C(openssl genrsa ...). Elliptic curve keys can be created with C(openssl ecparam -genkey ...)."
```
The correct way to reference modules is with `M(...)` with the FQCN in the brackets (i.e. here `community.crypto.openssl_privatekey`).

Reply:
I believe my latest commit should beet your requirements.

Reply:
Yep, that's perfect! Thanks! :)